packages/graphback-core/src/runtime/index.ts

```diff
@@ -4,4 +4,5 @@ export * from './GraphbackDataProvider'
 
 export * from "./CRUDService"
 export * from "./createCRUDService"
-export * from "./NoDataError"
\ No newline at end of file
+export * from "./NoDataError"
+export * from "./GraphbackProxyService"
```

Comment:
Hmm where does the ProxyService class come from?

Reply:
CI failure https://app.circleci.com/pipelines/github/aerogear/graphback/5195/workflows/47abd466-58b7-4997-9920-616de11ad610/jobs/5296/steps related.

Reply:
Yep.. It wasn't staged. My bad